src/leveled_cdb.erl

```diff
@@ -2396,6 +2435,16 @@ get_keys_byposition_manykeys_test_to() ->
     SampleList3 = cdb_getpositions(P2, KeyCount + 1),
     ?assertMatch(KeyCount, length(SampleList3)),
     
+    ?assertMatch(undefined, cdb_getcachedscore(P2, os:timestamp())),
+    ok = cdb_putcachedscore(P2, 80.0),
+    ?assertMatch(80.0, cdb_getcachedscore(P2, os:timestamp())),
+    timer:sleep(1000),
+    {NowMega, NowSecs, _} = Now = os:timestamp(),
+    ?assertMatch(80.0, cdb_getcachedscore(P2, Now)),
```

Comment:
here you can fake "os:timestamp" by remembering the old and add 1000 to its seconds.
No need to sleep for a second and make testing slower.
Or, what else is there purpose of the API with a timestamp?